src/webui/api/appcontroller.cpp

```diff
@@ -229,6 +229,7 @@ void AppController::preferencesAction()
     for (const Utils::Net::Subnet &subnet : asConst(pref->getWebUiAuthSubnetWhitelist()))
         authSubnetWhitelistStringList << Utils::Net::subnetToString(subnet);
     data["bypass_auth_subnet_whitelist"] = authSubnetWhitelistStringList.join("\n");
+    data["webUISessionTimeout"] = pref->getWebUISessionTimeout();
```

Comment:
The snake case is legacy style, see https://github.com/qbittorrent/qBittorrent/blob/master/CODING_GUIDELINES.md :
>For programming languages other than C++ (e.g. JavaScript) used in this repository and submodules, unless otherwise specified, coding guidelines listed here applies as much as possible.

Reply:
Look at the other `data` definitions case styles [appcontroller.cpp](https://github.com/qbittorrent/qBittorrent/blob/0fb9db69da741347961f09fc26829628752e160c/src/webui/api/appcontroller.cpp#L232), can you see that 😄

Reply:
Ok, changed, I'm afraid the existing convention is too strong...

Reply:
@glassez 
Could you please re-approve this PR? Here is the only change since your last review.